.github/dependabot.yml

```diff
@@ -4,3 +4,7 @@ updates:
     directory: "/dask_planner"
     schedule:
       interval: "daily"
+    ignore:
+      # arrow and datafusion are bumped manually
+      - dependency-name: "arrow"
```

Comment:
Are there any updates we would still want dependabot to look out for (i.e. patch version updates)?